src/lib/components/rectangle/figma.lock.ts

```diff
@@ -1,5 +1,5 @@
 export default {
   id: '1227:2688',
-  url: 'https://figma.com/file/PJ7OUutWRrSHW6qgaPNW1l/DS-betterplace-GENERAL?node-id=1227%3A2688',
+  url: 'https://www.figma.com/file/PJ7OUutWRrSHW6qgaPNW1l/Design-System-betterplace-GENERAL/file/PJ7OUutWRrSHW6qgaPNW1l/DS-betterplace-GENERAL?node-id=1227%3A2688',
```

Comment:
This looks incorrect

Reply:
Oops that was wrong in my .env file. It's fixed now.